src/components/Community/Tabs/Delegations/DelegateCardInfo.tsx

```
@@ -0,0 +1,493 @@
+						>
+							{contextHolder}
+							<CopyIcon className='-ml-[6px] scale-[60%] text-2xl text-lightBlue dark:text-icon-dark-inactive' />
+						</span>
+						{currentUserData && (
+							<ScoreTag
+								className='h-6 w-min px-[6px] py-1'
+								score={currentUserData}
+								iconWrapperClassName='mt-[5?.5px]'
+							/>
+						)}
+						{delegate?.dataSource && delegate?.dataSource?.length && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)